apps/meteor/ee/server/models/raw/LivechatRooms.ts

```diff
@@ -159,6 +168,7 @@ export class LivechatRoomsRawEE extends LivechatRoomsRaw implements ILivechatRoo
 		);
 	}
 
+	// TODO - check if removable, looks unused
```

Comment:
I think you're right here. This indeed doesn't seem to be used anywhere. Perhaps we can remove it?